### DISCUSSION TITLE
How to create Split button

### DISCUSSION THREAD

**Discussion by hoangnamitc** [orchidsoftware/platform#2336] [General]
**Is your feature request related to a problem? Please describe.**
I am create button save form
```
Button::make('Save')
  ->type(Color::SUCCESS())
  ->icon('save')
  ->method('save')
```
But i'm want create button multiple (save, save and new, save and close)
can use [Split button bootstrap](https://getbootstrap.com/docs/5.1/components/dropdowns/#split-button) to create it?
This platform support this?

**Describe the solution you'd like**
...

**Describe alternatives you've considered**
...

**Additional context**
...

Thank !

**Comment by czernika** [comment]
You may use Dropdown [field](https://orchid.software/en/docs/field/#dropdown)
